float32 x
float32 y
float32 z
float32 yaw
float32 position_thresh
float32 yaw_thresh
---
bool result